crate/src/jepsen/crate/lost_updates.clj

```diff
@@ -43,8 +45,8 @@
             (info node "Creating table sets")
```

Comment:
You may want to move this code to `setup!`, but I don't think this will break. :)